packages/app/src/app/pages/Dashboard/index.tsx

```diff
@@ -20,13 +20,14 @@ import { Sidebar } from './Sidebar';
 import { SIDEBAR_WIDTH } from './Sidebar/constants';
 import { Content } from './Content';
 import { WhatsNew } from './Components/WhatsNew/WhatsNew';
+import { NUOCT22 } from '../SignIn/Onboarding';
 
 const GlobalStyles = createGlobalStyle({
   body: { overflow: 'hidden' },
 });
 
 const COLUMN_MEDIA_THRESHOLD = 1600;
-const wnoct22 = 'wnoct22'; // = whats new oct 22
+const WNOCT22 = 'WNOCT22'; // = whats new oct 22
```

Comment:
Changing this won't retrigger the modal after this goes to prod?

Reply:
Oh, you're right 🧠! Let me undo this change.